docs/modules/ROOT/pages/cli_reference.adoc

```diff
@@ -70,6 +70,18 @@ Starts the operator in the cluster.
 
 include::cmd:ike[flags='serve --help --help-format=adoc']
 
+[#ike-create]
+=== `ike create`
+
+Creates or joins existing development session in the cluster. Allows to re-route cluster network traffic to your local process.
+
+The `create` command will use the `prepared-image` strategy when adding the reference to the `Session` and an Image is required.
```

Comment:
This sentence reads a bit odd - I don't know if it will use `prepared-image` strategy when I add `Session` and `Image` reference or that both are required?

Reply:
https://github.com/Maistra/istio-workspace/pull/252/commits/73b784145496295ab9df0f496eaa2d34d5c33bdf

Removed reference to Strategy for now until we doc that part